recipe_scrapers/albertheijn.py

```diff
@@ -3,9 +3,31 @@
 
 from ._abstract import AbstractScraper
 from ._utils import normalize_string
+from ._schemaorg import SchemaOrg
+
+import requests
+from bs4 import BeautifulSoup
+
+HEADERS = {
+    'Accept-Language': 'nl',  # ah.nl seems to block any requests not having both these headers.
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0'
+}
 
 
 class AlbertHeijn(AbstractScraper):
+    def __init__(self, url, proxies=None, timeout=None, *args, **kwargs):
```

Comment:
@Noxeus thanks for figuring this out!  Please could you also retain support for the [`html` argument](https://github.com/hhursev/recipe-scrapers/blob/cd0daddb5a52dafe40def2f1a06ee2b5b2957139/recipe_scrapers/_abstract.py#L34) in this scraper?